runlocal.sh

```diff
@@ -1,2 +1,2 @@
 #! /bin/sh
-bundle exec jekyll serve --config _config.yml,_config.local.yml
+bundle exec jekyll serve --config _config.yml,_config.local.yml $@
```

Comment:
This is ``ruby``. Sooooo versioning is craporino. Should we consider to use Docker to run it locally?